site/public/js/course-materials.js

```diff
@@ -0,0 +1,170 @@
+/* global IS_INSTRUCTOR, toggleCMFolders, toggleCMFolder */
+
+// eslint-disable-next-line no-unused-vars
+function setFolderCookie(folderPath, id, open) {
```

Comment:
It would be better to mark these as `exported` at the top of the file, similar to the `global` comment.

Reply:
I'm not sure what you mean.  The `no-unused-vars` error being ignored here is caused by the linter not seeing any usages of this function in this file.  This function is being used in a Twig file and there's no way for eslint to know that.  The solution is to tell eslint that it is used outside this file by marking it as "exported".